### ISSUE TITLE
Enable Self Hosted Runners For Public Repositories

### ISSUE THREAD

**Issue by kaozenn** [MinaProtocol/mina#12723]
The _Self Hosted Runners_ don't work for Public Repositories. We need to enable them to avoid _O1 Labs_ to be billed for GitHub Actions

Steps to Enable Self Hosted Runners:

1. In GitHub, can you go to the Organization Settings page
2. On the left hand side, Click Actions
3. Then, Runner Groups
4. Once on the Runner Groups page, Click the [Default](https://github.com/organizations/MinaProtocol/settings/actions/runner-groups/1) (I assume the link is correct)
5. Tick the Box "Allow public repositories"

![Image](https://user-images.githubusercontent.com/12220897/221341887-830bc671-e2fb-4973-91bd-765540bb84ad.png)

![Image](https://user-images.githubusercontent.com/12220897/221341889-287225ec-2556-440b-87dc-634315d801f3.png)

cc: @michal0mina

**Comment by kaozenn** [comment]
Hi @bkase ,
As @aneesharaines is on PTO, could you please take a look at this. It is a 5min task and it will prevent the GitHub account to be billed.
Thank you

**Comment by joaosreis** [comment]
Hi!
@aneesharaines @bkase do you have an update on this matter? Our team would like to use GitHub actions to generate some metrics for the product team, but we are currently blocked on this.

**Comment by kaozenn** [comment]
This is no longer relevant We can work against the [MinaFoundation](https://github.com/orgs/MinaFoundation) Repository
